src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/AsyncMethodBuilder.cs

```diff
@@ -191,28 +195,25 @@ private void NotifySynchronizationContextOfCompletion()
     /// </remarks>
     public struct AsyncTaskMethodBuilder
     {
-        /// <summary>A cached VoidTaskResult task used for builders that complete synchronously.</summary>
-#if PROJECTN
-        private static readonly Task<VoidTaskResult> s_cachedCompleted = AsyncTaskCache.CreateCacheableTask<VoidTaskResult>(default(VoidTaskResult));
-#else
-        private static readonly Task<VoidTaskResult> s_cachedCompleted = AsyncTaskMethodBuilder<VoidTaskResult>.s_defaultResultTask;
-#endif
-
-        /// <summary>The generic builder object to which this non-generic instance delegates.</summary>
-        private AsyncTaskMethodBuilder<VoidTaskResult> m_builder; // mutable struct: must not be readonly. Debugger depends on the exact name of this field.
```

Comment:
This is commented `Debugger depends on the exact name of this field.`

Not sure if changing it to `m_task` will cause issue?

Reply:
> Not sure if removing the wrapping and changing it to m_task will cause issue?

It might.  You would need to test it.  Does stepping out of an async method that yielded continue to work correctly?  Does stepping over a yielding await continue to work correctly?